package.json

```diff
@@ -13,7 +13,6 @@
     "type": "git",
     "url": "https://github.com/lynnaloo/oran.git"
   },
-  "pre-commit":["test"],
```

Comment:
Why did you delete pre-commit?  We added this to force you to pass your tests _which is eslint . --ext .js_  before you commit.  I think your changes are behind master

What you may need to do is add this repo _not your fork_ as a remote 
`git add remote lynnaloo git@github.com:lynnaloo/oranjs.git` then in your master branch git pull linda master.

After you do that, switch back to your branch in your fork and git rebase master.  This will bring you up to date with all the changes and apply your changes you made.  @lynnaloo